python/GafferUI/LabelPlugValueWidget.py

```diff
@@ -49,15 +49,20 @@ class LabelPlugValueWidget( GafferUI.PlugValueWidget ) :
 
 	def __init__( self, plug, horizontalAlignment=GafferUI.Label.HorizontalAlignment.Left, verticalAlignment=GafferUI.Label.VerticalAlignment.Center, **kw ) :
 
-		GafferUI.PlugValueWidget.__init__( self, QtWidgets.QWidget(), plug, **kw )
+		if isinstance( plug, Gaffer.Plug ) :
+			plugs = { plug }
+		else :
+			plugs = plug or set()
```

Comment:
@johnhaddon, this seemed to be the most compatible form of this. As it needs to support `None`, `Gaffer.Plug` or `set`. I saw you'd use the "not plug" test, rather than "not set" test in your earlier migrations. Let me know if you can think of a better, more general approach to this.